src/components/VisualPickerOption/index.js

```diff
@@ -1,55 +1,57 @@
+/* eslint-disable react/prop-types */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import { isArray, isString } from 'util';
```

Comment:
don't use this instead for check array use native Array.isArray and for string use typeof === 'string'